internal/distro/fedora/pipelines.go

```diff
@@ -0,0 +1,670 @@
+	kickstartOptions := ostreeKickstartStageOptions(makeISORootPath(ostreeRepoPath), options.OSTree.Ref)
+	pipelines = append(pipelines, *anacondaTreePipeline(repos, installerPackages, kernelVer, archName, d.product, d.osVersion, "edge"))
+	isolabel := fmt.Sprintf(d.isolabelTmpl, archName)
+	pipelines = append(pipelines, *bootISOTreePipeline(kernelVer, archName, d.vendor, d.product, d.osVersion, isolabel, kickstartOptions, payloadStages))
+	pipelines = append(pipelines, *bootISOPipeline(t.Filename(), d.isolabelTmpl, archName, false))
+	return pipelines, nil
+}
+
+func iotCorePipelines(t *imageType, customizations *blueprint.Customizations, options distro.ImageOptions, repos []rpmmd.RepoConfig, packageSetSpecs map[string][]rpmmd.PackageSpec) ([]osbuild.Pipeline, error) {
+	pipelines := make([]osbuild.Pipeline, 0)
+	pipelines = append(pipelines, *buildPipeline(repos, packageSetSpecs[buildPkgsKey], t.arch.distro.runner))
+
+	treePipeline, err := osPipeline(t, repos, packageSetSpecs[osPkgsKey], packageSetSpecs[blueprintPkgsKey], customizations, options, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	pipelines = append(pipelines, *treePipeline)
+	pipelines = append(pipelines, *ostreeCommitPipeline(options, t.arch.distro.osVersion))
+
+	return pipelines, nil
+}
+
+func edgeCommitPipelines(t *imageType, customizations *blueprint.Customizations, options distro.ImageOptions, repos []rpmmd.RepoConfig, packageSetSpecs map[string][]rpmmd.PackageSpec, rng *rand.Rand) ([]osbuild.Pipeline, error) {
+	pipelines, err := iotCorePipelines(t, customizations, options, repos, packageSetSpecs)
+	if err != nil {
+		return nil, err
+	}
+	tarPipeline := osbuild.Pipeline{
+		Name:  "commit-archive",
+		Build: "name:build",
+	}
+	tarPipeline.AddStage(tarStage("ostree-commit", t.Filename()))
+	pipelines = append(pipelines, tarPipeline)
+	return pipelines, nil
+}
+
+func iotContainerPipelines(t *imageType, customizations *blueprint.Customizations, options distro.ImageOptions, repos []rpmmd.RepoConfig, packageSetSpecs map[string][]rpmmd.PackageSpec, rng *rand.Rand) ([]osbuild.Pipeline, error) {
+	pipelines, err := iotCorePipelines(t, customizations, options, repos, packageSetSpecs)
+	if err != nil {
+		return nil, err
+	}
+
+	nginxConfigPath := "/etc/nginx.conf"
+	httpPort := "8080"
+	pipelines = append(pipelines, *containerTreePipeline(repos, packageSetSpecs[containerPkgsKey], options, customizations, nginxConfigPath, httpPort))
+	pipelines = append(pipelines, *containerPipeline(t, nginxConfigPath, httpPort))
+	return pipelines, nil
+}
+
+func buildPipeline(repos []rpmmd.RepoConfig, buildPackageSpecs []rpmmd.PackageSpec, runner string) *osbuild.Pipeline {
+	p := new(osbuild.Pipeline)
+	p.Name = "build"
+	p.Runner = runner
+	p.AddStage(osbuild.NewRPMStage(rpmStageOptions(repos), osbuild.NewRpmStageSourceFilesInputs(buildPackageSpecs)))
+	p.AddStage(osbuild.NewSELinuxStage(selinuxStageOptions(true)))
+	return p
+}
+
+func osPipeline(t *imageType,
```

Comment:
I can see the line you mention about GPG keys, perhaps I added it back in one of my updates....

Reply:
I'll resolve the conversation since the line you mention is already included...
https://github.com/osbuild/osbuild-composer/blob/3b8fd155956b75d7495185d33681b8f13968ca37/internal/distro/fedora/pipelines.go#L258-L260